src/vm.rs

```diff
@@ -157,6 +162,7 @@ impl From<VmPagesError> for EcallError {
     fn from(error: VmPagesError) -> EcallError {
         match error {
             VmPagesError::PageFault(pf, e, addr) => EcallError::PageFault(pf, e, addr),
+            VmPagesError::NotEnoughMemory => EcallError::Sbi(SbiError::InvalidParam),
```

Comment:
This should also follow the same TODO about error code mapping form GuestTrackingError.